packages/atomic/src/components/search-box-suggestions/atomic-search-box-recent-queries/atomic-search-box-recent-queries.tsx

```diff
@@ -122,16 +122,18 @@ export class AtomicSearchBoxRecentQueries {
       key: 'recent-query-clear',
       content: (
         <div class="flex justify-between w-full">
-          <span class="font-bold">
+          <span class="font-bold" part="recent-query-title">
             {this.bindings.i18n.t('recent-searches')}
           </span>
-          <span>{this.bindings.i18n.t('clear')}</span>
+          <span part="recent-query-clear">{this.bindings.i18n.t('clear')}</span>
         </div>
       ),
       onSelect: () => {
         this.recentQueriesList.clear();
         this.bindings.triggerSuggestions();
       },
+      part: 'recent-query-header',
+      hideIf: (isLast: boolean) => isLast,
```

Comment:
We should atomically handle that case, it seems like a bug, maybe we can improve the condition on line 113, and remove that prop too.

Reply:
I agree with JF on this, I don't think we should add this prop.

IMO, if people want to hide a suggestion, I think they should just not return it.

Reply:
I thought about it, but I think at this point we have access to this only
![Screenshot 2022-05-11 at 19 14 49](https://user-images.githubusercontent.com/30511433/167918281-1f44eca3-bc41-468e-a628-445d46a2ef3a.png)
and not the actual elements that would be rendered. Do you think it's worth adding it to the state?

Reply:
I guess one thing that we could do is, in `atomic-search-box`, check that the last element doesn't have the `recent-query-clear`. that doesn't expose any of this functionality. what do you think? @ThibodeauJF